src/components/tutorials/advanced-tutorial.js

```diff
@@ -153,7 +126,7 @@ class AdvancedTutorial extends React.Component {
               verb='GET'
               url='/submissions/:id/score' />
             <Advanced5 />
-            <p>A few things have changed - the final score increased to 15.5. We know this change is due to our PATCH by looking at the score component contributed by <code>ACI_HIE_1</code> - it increased from 1 to 5, since the proportion increased from 10 to 50 out of 100 with ten possible points from this measure. The ACI base score went up from 58 to 62, and with the ACI component being 25% of the score our final score increased by 1.</p>
+            <p>A few things have changed - the final score decreased from 7.5 to 15. We know this change is due to our PATCH by looking at the score component for IA. Since we changed IA_EPA_4 to false, it is no longer considered for scoring.</p>
```

Comment:
whoops this is wrong. should be:

 <p>A few things have changed - the final score decreased from 15 to 7.5. We know this change is due to our PATCH by looking at the score component for IA. Since we changed IA_EPA_4 to false, it is no longer considered for scoring.</p>